packages/hooks/use-infinite-scroll/src/index.ts

```diff
@@ -7,12 +7,12 @@ import {
 } from "@yamada-ui/utils"
 import { useCallback, useEffect, useMemo, useRef, useState } from "react"
 
-const isScrollable = (el: HTMLElement, isVertical: boolean) => {
+const scrollable = (el: HTMLElement, vertical: boolean) => {
```

Comment:
It would be better not to omit the `is` in the function, because you don't know what it returns.

It is not clear what the function is supposed to return.For example, `scrollable` would be associated with making the passed element scrollable.

Reply:
Sure! I agree on that! I will keep that in mind!

I have revert it back, please re-review :)